### DISCUSSION TITLE
append HTML to an element

### DISCUSSION THREAD

**Discussion by ivanjeremic** [cheeriojs/cheerio#2215] [Q&A] [answered]
I want to append HTML to my body without deleting existing elements, I tried this:

```ts
$("body").html( `<div>${name}</div>`).html();
```

but it has not the result I want. Can someone please help me? Thanks

**Comment by fb55** [comment, marked as answer]
Have a look at [append](https://cheerio.js.org/classes/Cheerio.html#append)!
